### ISSUE TITLE
~ Feature Request ~ Displaying 1st few letters for context

### ISSUE THREAD

**Issue by VerAn1015** [DearVikki/obsidian-cloze-plugin#13]
Hi! First, thank you for developing this simple yet very useful plugin! 

I would like to suggest a toggleable feature that shows the first letter (or a percentage of the entire cloze say for example, 20% of a cloze containing 10 letters would show me 2 of its 1st letters). I have been using this plugin for more than a month now, and I have an overwhelming amount of clozes in a note since I am a med student and I believe this feature would reduce that friction and would be helpful when still trying to digest new information.

Again, thank you so much and I appreciate the effort you put into developing this plugin! Happy Holidays!

**Comment by DearVikki** [comment]
Hi @VerAn1015, thank you so much for your kind words and for using it! :D

I really appreciate your suggestion for a toggleable feature that shows the first letter or a percentage of the entire cloze. We have a Hint feature, but it only applys to custom Cloze and required manual input. Your suggestion aligns perfectly with Hint and I'll definitely consider adding this feature!

Before I proceed, may I ask which type of Cloze you use most frequently in your scenario? Is it the custom-added ones or the auto-transformed ones? This information will help me determine the best approach/priority for enhancing the plugin. If you use custom Cloze, I can focus on enhancing the Hint feature for this specific type, which would be a simpler and quicker solution. Otherwise, I may have to explore implementing the Hint feature for all kinds of Clozes, which may require more modifications, (but it definitely is an inevitable path that I must take in the end, haha!😆)

Thank you again for your valuable feature request, and may you an early happy new year! 🎉 🥂

**Comment by VerAn1015** [comment]
Hello @DearVikki. Thank you for the response! Unfortunately, I use the latter more often as it saves me more time haha 😅

Nonetheless, thank you very much for considering this feature request, may you have a great year as well! 🎉

**Comment by DearVikki** [comment]
Hi @VerAn1015. New year, new feature! Here comes the hint feature in the latest release(version 0.1.12) :D

You can set the hint strategy in the setting now:
![image](https://github.com/DearVikki/obsidian-cloze-plugin/assets/18145936/0c8bc82a-6fa9-4a85-a8bc-4c205fd33e70)

And you may also right click the hidden cloze to dynamically reveal more content of the cloze! (I'm not sure if its a common scenario or not, but I just thought it would be nice to have it. 😆 )

This feature is still in beta, so if you come across any problems or have ideas for making it even better, please let me know. I'd really appreciate your feedback! Thank you so much. 🍺

**Comment by VerAn1015** [comment]
Wow @DearVikki 🫨! Words are not enough to express how grateful I am right now. I expected it to take some time (as implied by your previous replies) to have this feature, but in less than a month?! You are just amazing! A life changing feature for me. It works exactly as how I imagined it to be. You added not just one but even both of the options I suggested and even thought of the "More hint" feature which is an amazing addition as well! 🫨 This really made my day. A Happy New Year indeed!  Thank you so much and I appreciate the effort you put into this!!!
